scripts/instance/mask_rcnn/train_mask_rcnn.py

```diff
@@ -31,6 +32,11 @@
 except ImportError:
     hvd = None
 
+try:
+    from mpi4py import MPI
+except ImportError:
```

Comment:
please advice MPI is not found, and how it can be installed

Reply:
we are actually not requiring the user to install MPI. It is optional. The script would still work, albeit slower during validation.